modules/RTC/RTCUtils.js

```diff
@@ -321,6 +321,7 @@ function pollForAvailableMediaDevices() {
  * @emits RTCEvents.DEVICE_LIST_CHANGED
  */
 function onMediaDevicesListChanged(devices) {
+    var initialDeviceList = currentlyAvailableMediaDevices.length == 0;
```

Comment:
Ok I got that, but even when this is available we can implement something similar. 
And what about the output device, logging? This also solves the problem, when to send to callstats the inital output device data, what is your proposal, when we should do that if we merge PR#139?